x-pack/plugins/alerting/server/saved_objects/migrations.test.ts

```diff
@@ -970,6 +970,26 @@ describe('successful migrations', () => {
       });
     });
   });
+
+  describe('7.14.1', () => {
+    test('security solution author field is migrated to array', () => {
```

Comment:
maybe a test to make sure existing author fields aren't overridden?

Reply:
Oh thank you, I will add that.

Reply:
Added here: https://github.com/elastic/kibana/pull/107230/commits/f507b65603db0271f7f2c22fb11ed2366dc68020